### ISSUE TITLE
Error: message is not defined

### ISSUE THREAD

**Issue by jermainedilao** [DavideViolante/pr-reviews-reminder-action#71]
```
Getting open pull requests...
There are 14 open pull requests
There are 8 pull requests waiting for reviews
Error: message is not defined
```

I am always getting this error in the logs. 

Can you help me?

**Comment by DavideViolante** [comment]
Fixed in v2.1.1, please use that in the action.

**Comment by jermainedilao** [comment]
It's working now, thanks!
